level_handler.go

```diff
@@ -271,10 +271,15 @@ func (s *levelHandler) get(key []byte) (y.ValueStruct, error) {
 			continue
 		}
 
-		it := th.NewIterator(false)
+		it, ok := th.PointGet(key)
+		if !ok {
+			it = th.NewIterator(false)
+			it.Seek(key)
+		} else if it == nil {
+			continue
+		}
 		defer it.Close()
```

Comment:
I have just noticed, is this `defer it.Close()` useless? 

Because we have increment table's reference count in `getTableForKey()`, we can close this iterator directly when step to search next table? 

Or even further we can allocate a new `Iterator` without increment table's counter? This can reduce an expensive atomic write.

@coocood